modules/xmpp/JingleSessionPC.js

```diff
@@ -954,7 +954,15 @@ export default class JingleSessionPC extends JingleSession {
                 // FIXME we may not care about RESULT packet for session-accept
                 // then we should either call 'success' here immediately or
                 // modify sendSessionAccept method to do that
-                this.sendSessionAccept(success, failure);
+                this.sendSessionAccept(() => {
+                    success();
+
+                    this.room.eventEmitter.emit(XMPPEvents.SESSION_ACCEPT, this);
+                }, () => {
+                    failure();
```

Comment:
how is the error handled here?

Reply:
I don't know, this is the callback that was executed before.

Reply:
I had added the error to the callback.